src/css/doc.css

```
@@ -147,6 +147,16 @@
   margin-top: var(--base-large-space);
 }
 
+/**
```

Don't think we need two `*` for multi line comment, just one no?
```suggestion
/*
```

---

Mmm. Seems to be some sort of IntelliJ preference. Yup; it's fine with just the single asterisk.